### ISSUE TITLE
failed MSpanList_Insert 0x604000 0x4233b162b71ea 0x0

### ISSUE THREAD

**Issue by adamgotterer** [lafikl/telsocket#8]
```
telsocket -url ws://127.0.0.1:9222/devtools/
failed MSpanList_Insert 0x604000 0x4233b162b71ea 0x0
fatal error: MSpanList_Insert

runtime stack:
runtime.throw(0x42d22b)
        /usr/local/go/src/runtime/panic.go:491 +0xad fp=0x7ffeefbff350 sp=0x7ffeefbff320
runtime.MSpanList_Insert(0x453208, 0x604000)
        /usr/local/go/src/runtime/mheap.c:692 +0x8f fp=0x7ffeefbff378 sp=0x7ffeefbff350
MHeap_FreeSpanLocked(0x44fe00, 0x604000, 0x100)
        /usr/local/go/src/runtime/mheap.c:583 +0x163 fp=0x7ffeefbff3b8 sp=0x7ffeefbff378
MHeap_Grow(0x44fe00, 0x8, 0x0)
        /usr/local/go/src/runtime/mheap.c:420 +0x1a8 fp=0x7ffeefbff3f8 sp=0x7ffeefbff3b8
MHeap_AllocSpanLocked(0x44fe00, 0x1, 0x0)
        /usr/local/go/src/runtime/mheap.c:298 +0x365 fp=0x7ffeefbff438 sp=0x7ffeefbff3f8
mheap_alloc(0x44fe00, 0x1, 0x12, 0x0)
        /usr/local/go/src/runtime/mheap.c:190 +0x121 fp=0x7ffeefbff460 sp=0x7ffeefbff438
runtime.MHeap_Alloc(0x44fe00, 0x1, 0x10000000012, 0xdca9)
        /usr/local/go/src/runtime/mheap.c:240 +0x66 fp=0x7ffeefbff498 sp=0x7ffeefbff460
MCentral_Grow(0x457b78, 0x0)
        /usr/local/go/src/runtime/mcentral.c:197 +0x8b fp=0x7ffeefbff500 sp=0x7ffeefbff498
runtime.MCentral_CacheSpan(0x457b78, 0x0)
        /usr/local/go/src/runtime/mcentral.c:85 +0x167 fp=0x7ffeefbff538 sp=0x7ffeefbff500
runtime.MCache_Refill(0x600000, 0x12, 0x0)
        /usr/local/go/src/runtime/mcache.c:90 +0xa0 fp=0x7ffeefbff560 sp=0x7ffeefbff538
runtime.mcacheRefill_m()
        /usr/local/go/src/runtime/malloc.c:368 +0x57 fp=0x7ffeefbff580 sp=0x7ffeefbff560
runtime.onM(0x35f278)
        /usr/local/go/src/runtime/asm_amd64.s:273 +0x9a fp=0x7ffeefbff588 sp=0x7ffeefbff580
runtime.mallocgc(0x120, 0x2c9000, 0x0, 0x0)
        /usr/local/go/src/runtime/malloc.go:178 +0x849 fp=0x7ffeefbff638 sp=0x7ffeefbff588
runtime.newobject(0x2c9000, 0x600000)
        /usr/local/go/src/runtime/malloc.go:353 +0x49 fp=0x7ffeefbff660 sp=0x7ffeefbff638
runtime.newG(0x2767a)
        /usr/local/go/src/runtime/proc.go:233 +0x2a fp=0x7ffeefbff678 sp=0x7ffeefbff660
allocg(0x440900)
        /usr/local/go/src/runtime/proc.c:925 +0x1f fp=0x7ffeefbff688 sp=0x7ffeefbff678
runtime.malg(0x8000, 0x440ac0)
        /usr/local/go/src/runtime/proc.c:2106 +0x1f fp=0x7ffeefbff6b8 sp=0x7ffeefbff688
runtime.mpreinit(0x4410e0)
        /usr/local/go/src/runtime/os_darwin.c:137 +0x27 fp=0x7ffeefbff6d0 sp=0x7ffeefbff6b8
mcommoninit(0x4410e0)
        /usr/local/go/src/runtime/proc.c:201 +0xc9 fp=0x7ffeefbff6f8 sp=0x7ffeefbff6d0
runtime.schedinit()
        /usr/local/go/src/runtime/proc.c:138 +0x55 fp=0x7ffeefbff720 sp=0x7ffeefbff6f8
runtime.rt0_go(0x7ffeefbff758, 0x3, 0x7ffeefbff758, 0x0, 0x0, 0x3, 0x7ffeefbff928, 0x7ffeefbff932, 0x7ffeefbff937, 0x0, ...)
        /usr/local/go/src/runtime/asm_amd64.s:95 +0x116 fp=0x7ffeefbff728 sp=0x7ffeefbff720
```

**Comment by ppamorim** [comment]
I am getting this same issue on macOS Mojave.

**Comment by sergiotudela** [comment]
Same here.

**Comment by eleftrik** [comment]
Same here

**Comment by etherviz** [comment]
ditto

**Comment by amalkh5** [comment]
fixed in [v1.0.1](https://github.com/lafikl/telsocket/releases/tag/v1.0.1)
